src/Orleans/Core/ClientBuilder.cs

```diff
@@ -86,6 +87,8 @@ private static void AddBasicServices(IServiceCollection services, ClientConfigur
             services.AddSingleton<StreamProviderManager>();
             services.AddSingleton<ClientStatisticsManager>();
             services.AddFromExisting<IStreamProviderManager, StreamProviderManager>();
+            services.AddFromExisting<IStreamProviderRuntime, ClientProviderRuntime>();
+            services.AddSingleton<IStreamSubscriptionManagerAdmin>(sp => new StreamSubscriptionManagerAdmin(sp.GetRequiredService<IStreamProviderRuntime>()));
```

Comment:
I don't think you need to use the service provider to look up the runtime, constructor injection should do this for you with:
services.AddSingleton<IStreamSubscriptionManagerAdmin,StreamSubscriptionManagerAdmin>();